pages/setup/screen-readers/nvda/README.md

```diff
@@ -6,7 +6,7 @@ changed: "2020-04-07"
 
 # NVDA installation and configuration
 
-**NVDA (Non Visual Desktop Access) is a lightweight and reliable open source screen reader. It is very standards-compliant and is the best choice to be used as a reference for accessibility. NVDA is the perfect companion to be fired up frequently while developing. After using it for a while, you will respect it as a firm but fair teacher.**
+**NVDA (Non Visual Desktop Access) is a lightweight and reliable open source desktop screen reader. It is very standards-compliant and is the best choice to be used as a reference for accessibility. NVDA is the perfect companion to be fired up frequently while developing. After using it for a while, you will respect it as a firm but fair teacher.**
```

Comment:
As NVDA is the most used desktop screen reader today, I suggest rephrasing a bit:

> NVDA (Non Visual Desktop Access) is the most-used desktop screen reader. As it is Open Source, reliable and very standards-compliant it is the best choice to be used as a reference for accessibility...

PS: NVDA sadly isn't as lightweight anymore as it used to be... ☹️